spec/services/youtube_service_spec.rb

```diff
@@ -15,7 +15,7 @@
 
     it "playlist_items_info" do
       playlist_items_info = @service.playlist_items_info(@id)
-      expect(playlist_items_info[:items].length).to eq(13)
+      expect(playlist_items_info[:items].length).to eq(10)
```

Comment:
This kept failing. the number of videos in the playlist has already jumped to 15. I changed it to a turning playlist that has 10 vidoes